.github/renovate.json

```diff
@@ -3,27 +3,26 @@
   "extends": [
     "github>SocialGouv/renovate-config"
   ],
+  "schedule": ["after 10pm and before 5am on every weekday", "every weekend"],
```

Comment:
use extends [`schedule:nonOfficeHours`](https://docs.renovatebot.com/presets-schedule/#schedulenonofficehours)

Reply:
how do you merge it with every month ?